src/components/CurrentRefinements/CurrentRefinements.tsx

```diff
@@ -1,14 +1,30 @@
 import React from 'preact-compat';
-import PropTypes from 'prop-types';
 import { isSpecialClick, capitalize } from '../../lib/utils';
+import { Item } from '../../connectors/current-refinements/connectCurrentRefinements';
+import { CurrentRefinementsComponentCSSClasses } from '../../widgets/current-refinements/current-refinements';
 
-const createItemKey = ({ attribute, value, type, operator }) =>
+type CurrentRefinementsProps = {
+  items: Item[];
+  cssClasses: CurrentRefinementsComponentCSSClasses;
+};
+
+const createItemKey = ({
+  attribute,
+  value,
+  type,
+  operator,
+}: {
+  attribute: string;
+  value: string;
+  type: string;
+  operator?: string;
+}): string =>
   [attribute, type, value, operator]
     .map(key => key)
     .filter(Boolean)
     .join(':');
 
-const handleClick = callback => event => {
+const handleClick = (callback: () => void) => (event: any) => {
```

Comment:
we might be able to inline this, instead of this HOF

Reply:
That's a pattern we use [elsewhere](https://github.com/algolia/instantsearch.js/blob/bc0f9e245f28fe7ee6f9578c86ae833607d36145/src/components/Pagination/Pagination.js#L102-L110). We can refactor this later.